packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1849,7 +1849,7 @@ class ImportedVpc extends VpcBase {
 
 class LookedUpVpc extends VpcBase {
   public readonly vpcId: string;
-  public readonly internetConnectivityEstablished: IDependable = new ConcreteDependable();
+  public readonly internetConnectivityEstablished: IDependable = new DependencyGroup();
```

Comment:
similar. can we mark `internetConnectivityEstablished` as deprecated and introduce new equivalent props?